Development.md

```diff
@@ -1,24 +1,35 @@
-# Local Testing
+# Development
+
+## Contribute
+
+Before you submit your PRs, please ensure no error with following command:
+
+* `make presubmit`
+* `make test`
+
+To do more comprehensive testing, please refer to [Local Testing](#local-testing).
+
+## Local Testing
 
 Along with running in a Kubernetes, ztunnel can be run locally for development purposes.
 
 This doc covers ztunnel specifically, for general Istio local development see
 [Local Istio Development](https://github.com/howardjohn/local-istio-development).
 
-## Local overrides
+### Overrides
 
 There are a variety of config options that can be used to replace components with mocked ones:
 
-* `FAKE_CA="true"` - this will use self-signed fake certificates, eliminating a dependency on a CA
-* `XDS_ADDRESS=""` - disables XDS client completely
-* `LOCAL_XDS_PATH=./examples/localhost.yaml` - read XDS config from a file.
-  This example adds a workload for `127.0.0.1`, allowing us to send requests to/from localhost.
-* `NODE_NAME=local` - configures which node the ztunnel is running as.
-  This impacts the networking path of requests. In the `localhost.yaml` example, `NODE_NAME=local` would make localhost use the in-memory fast path; without it HBONE would be used.
+* `FAKE_CA="true"`: this will use self-signed fake certificates, eliminating a dependency on a CA
+* `XDS_ADDRESS=""`: disables XDS client completely
+* `LOCAL_XDS_PATH=./examples/localhost.yaml`: read XDS config from a file.
+* `NODE_NAME=local`: configures which node the ztunnel is running as.This impacts the networking path of requests
```

Comment:
This was just dropped with https://github.com/istio/ztunnel/pull/915

so you can remove the NODE-NAME stuff entirely.

Reply:
Done.